.github/workflows/org-gate.yml

```
@@ -0,0 +1,116 @@
+name: org-gate
```

actionlint (1.7.12)

[error] "on" section is missing in workflow

(syntax-check)